litellm/proxy/proxy_server.py

```diff
@@ -1475,7 +1475,7 @@ async def load_team_config(self, team_id: str):
             if "team_id" not in team:
                 raise Exception(f"team_id missing from team: {team}")
             if team_id == team["team_id"]:
-                team_config = team
+                team_config = copy.deepcopy(team)
```

Comment:
hey @Manouchehri sorry just seeing this - do you know what was causing this?

Reply:
I guess this solution does not provoke errors but the logging does not work